client/layout/masterbar/logged-in.jsx

```diff
@@ -639,8 +688,13 @@ export default connect(
 			isSiteMigrationActiveRoute( state );
 
 		const siteCount = getCurrentUserSiteCount( state ) ?? 0;
-		const { shouldShowGlobalSidebar } = useGlobalSidebar( siteId, sectionGroup );
-
+		const { shouldShowGlobalSidebar } = useGlobalSidebar( currentSelectedSiteId, sectionGroup );
```

Comment:
![badge](https://img.shields.io/badge/review-nits-green.svg)
👋 
I suggest using a selector here because a hook cannot be utilized in the `connect()` function. Please refer to https://github.com/Automattic/wp-calypso/pull/87749 🙂